submissions/kikinovk/Memory_Pair-game/js/main.js

```diff
@@ -0,0 +1,123 @@
+const images = [
+  { src : './img/card_1.svg',
+    id : '01'
+  },
+  { src : './img/card_2.svg',
+    id : '02'
+  },
+  { src : './img/card_3.svg',
+      id : '03'
+  },
+  { src : './img/card_4.svg',
+    id : '04'
+  },
+  { src : './img/card_5.svg',
+    id : '05'
+  },
+  { src : './img/card_6.svg',
+    id : '06'
+  }
+];
+
+let openCard;
+const delay = 500;
+const maxCard = 6;
+const cards = [];
+const gameBoard = document.querySelector('.gameboard');
+const restartButton = document.querySelector('#restart_game');
+
+const selectImage = (cards, images) => {
+  let maxRandom = images.length;
+  let random = 0;
+
+  do {
+    random = Math.floor(Math.random() * Math.floor(maxRandom));
+  } while (cards
+              .filter(card => card.dataset.id === images[random].id)
+              .length === 2);
+  return images[random]
+};
+
+const creatCard = ({src, id}) => {
+  let card =  document.createElement('div');
+
+  card.classList.add('card');
+  card.dataset.id = id;
+  card.innerHTML = `<div class="card__front">
+                      <img src="./img/face.png" class="card__img--front" >
+                    </div>
+                    <div class="card__back">
+                      <img src="${src}" class="card__img--back">
```

Comment:
alt

Reply:
What did you mean?

Reply:
[Html alt attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attributes)